gensim/models/ldamodel.py

```diff
@@ -82,53 +98,70 @@ def update_dir_prior(prior, N, logphat, rho):
 
 
 class LdaState(utils.SaveLoad):
-    """
-    Encapsulate information for distributed computation of LdaModel objects.
+    """Encapsulate information for distributed computation of LdaModel objects.
 
     Objects of this class are sent over the network, so try to keep them lean to
     reduce traffic.
 
     """
 
     def __init__(self, eta, shape, dtype=np.float32):
+        """Initialize the state.
```

Comment:
no need to write "Initialize the state." always, this is obvious (here and everywhere)